docs/components/stories.md

```diff
@@ -0,0 +1,95 @@
+# Component stories
+
+## Setup
+
+TODO
+
+## Component configuration
+
+The Storybook configuration is a module which exports properties about the component to be rendered.
+
+Property       | Description                                               | Required
+---------------|-----------------------------------------------------------|----------
+`component`    | A reference to the top-level function to render           | Yes
+`package`      | The component's package manifest                          | Yes
+`dependencies` | An object containing the Origami dependencies to load     | No
+`stories`      | An array of story configuration modules                   | Yes
+`knobs`        | A module generating dynamically editable knobs            | No
+
+Here is an example component configuration:
+
+```js
+const { Denshiba } = require('../');
+
+// Reference to top-level function to render
+exports.component = Denshiba;
+
+// The component's package manifest
+exports.package = require('../package.json');
+
+// Origami dependencies to load (from the Build Service)
+exports.dependencies = {
+	'o-normalise': '^1.6.0',
+	'o-typography': '^5.5.0'
+};
+
+// Story configuration modules
+exports.stories = [
+	require('./food')
+];
+
+// A module generating dynamically editable knobs
+exports.knobs = require('./knobs');
```

Comment:
Worth mentioning the [fetch-mock stuff](https://github.com/Financial-Times/x-dash/pull/109/files) here too? Or too niche?

Reply:
Yes I think so, but I'll let your team write it 😄

Reply:
👍